audio_service/android/build.gradle

```diff
@@ -38,6 +38,6 @@ android {
 }
 
 dependencies {
-    implementation 'androidx.core:core:1.3.2'
-    implementation 'androidx.media:media:1.2.0'
```

Comment:
Bumping this up can cause crashes if users use not right gradle tools, see https://stackoverflow.com/a/62969918/9710294 for details.

We should probably mention this in readme/faq

Reply:
media:1.2.0 already supports onSetPlaybackSpeed so we don't need to deal with this problem yet.

Reply:
New versions had a few bugfixes and I though it would be useful to bump it now, might have been confusing as part of this PR though

Reply:
Reverted it and filed https://github.com/ryanheise/audio_service/issues/752